
2024-04-25 12:28:08.914199: I external/local_tsl/tsl/platform/default/subprocess.cc:304] Start cannot spawn child process: No such file or directory
2024-04-25 12:28:14.254840: W tensorflow/core/common_runtime/type_inference.cc:339] Type inference failed. This indicates an invalid graph that escaped type checking. Error message: INVALID_ARGUMENT: expected compatible input types, but input 1:
type_id: TFT_OPTIONAL
args {
  type_id: TFT_PRODUCT
  args {
    type_id: TFT_TENSOR
    args {
      type_id: TFT_INT32
    }
  }
}
 is neither a subtype nor a supertype of the combined inputs preceding it:
type_id: TFT_OPTIONAL
args {
  type_id: TFT_PRODUCT
  args {
    type_id: TFT_TENSOR
    args {
      type_id: TFT_FLOAT
    }
  }
}
	for Tuple type infernce function 0
	while inferring type of node 'cond_36/output/_23'
2024-04-25 12:28:15.139872: I external/local_xla/xla/service/service.cc:168] XLA service 0x7f77dec32c00 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2024-04-25 12:28:15.139927: I external/local_xla/xla/service/service.cc:176]   StreamExecutor device (0): NVIDIA GeForce RTX 4070 Laptop GPU, Compute Capability 8.9
2024-04-25 12:28:15.146116: I tensorflow/compiler/mlir/tensorflow/utils/dump_mlir_util.cc:269] disabling MLIR crash reproducer, set env var `MLIR_CRASH_REPRODUCER_DIRECTORY` to enable.
2024-04-25 12:28:15.160758: I external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:454] Loaded cuDNN version 8907
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1714040895.221818  408168 device_compiler.h:186] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.



 16/151 [==>...........................] - ETA: 1:06 - loss: 6.8690
